test/e2e/common/node/container_probe.go

```diff
@@ -143,7 +143,7 @@ var _ = SIGDescribe("Probing container", func() {
 		livenessProbe := &v1.Probe{
 			ProbeHandler:        execHandler([]string{"cat", "/tmp/health"}),
 			InitialDelaySeconds: 15,
-			TimeoutSeconds:      5, // default 1s can be pretty aggressive in CI environments with low resources
+			TimeoutSeconds:      10, // default 1s can be pretty aggressive in CI environments with low resources
```

Comment:
is there a way to only increase this timeout for (dockershim) tests?

Reply:
there was a REMOTE_RUNTIME variable or something similar IIRC, I don't know if it is still used though

Reply:
will check about it